pkg/env/nats_config_test.go

```diff
@@ -1,134 +1,63 @@
-//go:build unit
-
 package env
 
 import (
-	"os"
-	"reflect"
-	"strings"
 	"testing"
-	"time"
+
+	"github.com/kyma-project/eventing-manager/test/utils"
+	"github.com/stretchr/testify/require"
 )
 
-func Test_GetNATSConfig(t *testing.T) {
```

Comment:
This test will be fixed as part of: [link](https://github.com/kyma-project/eventing-manager/pull/43/files#r1287032171)